bin/__tests__/getIntegrationAccessToken.test.js

```diff
@@ -13,7 +13,9 @@
 const proxyquire = require('proxyquire');
 
 const METASCOPES = [
+  'ent_reactor_sdk',
   'ent_reactor_extension_developer_sdk',
```

Comment:
Let's put a comment right above this metascope that says `// The below metascopes are necessary to maintain for integrations created before the `ent_reactor_sdk` metascope existed.`